### ISSUE TITLE
Imgbot broke JPEG

### ISSUE THREAD

**Issue by MichaIng** [imgbot/Imgbot#1270]
Just faced: https://github.com/MichaIng/DietPi-Docs/pull/794/files

See the Dropbear logo at the bottom, which has its bottom content removed.

**Comment by GrigoreMihai** [comment]
Hello, we will review this issue, but it would help if we would have a larger sample of images that are breaking to identify patterns. 

For now you can use our `ignoredFiles` option to exclude the image with the issue from optimization as mention here: https://imgbot.net/docs/#configuration .

**Comment by MichaIng** [comment]
For me it is as well the first JPEG I see breaking, so I have no other example. Out of interest, I saved the file again with GIMP, removing some metadata. If this works, it might give an indication.

**Comment by wojtekxtx** [comment]
Sorry for bumping, but just saw same result as @MichaIng. Some 10px from bottomline cropped from `jpg`.

**Comment by wojtekxtx** [comment]
Are you pertaining to me @MichaIng ?

**Comment by MichaIng** [comment]
Of course, GrigoreMihai asked for further examples. I have only one, so if you have another one, that might help. An "I have the same issue" comment doesn't help here, but the images themselves are needed to investigate why this happens 🙂.

**Comment by wojtekxtx** [comment]
@MichaIng due to its nature, I will not share JPGs publicly, sorry

**Comment by MTerrases22** [comment]
> Just faced: https://github.com/MichaIng/DietPi-Docs/pull/794/files
> 
> See the Dropbear logo at the bottom, which has its bottom content removed.

Trouble shooting will help , starting over, turning problematic objects off and on 9 times out of 10 the issues is no longer an issue.

**Comment by MichaIng** [comment]
> Trouble shooting will help , starting over, turning problematic objects off and on 9 times out of 10 the issues is no longer an issue.

I am open to help troubleshooting, but what do you mean with turning off and on the problematic object, in the context of using ImgBot?
